src/HasteModuleLoader/HasteModuleLoader.js

```diff
@@ -113,6 +112,9 @@ function _getCacheFilePath(config) {
 
 function Loader(config, environment, resourceMap) {
   this._config = config;
+  this._CoverageCollector = require(
+    config.testCollector || '../CoverageCollector'
```

Comment:
I wrote this because that one test passes an empty config and that config doesn't get merged with default config. So it fails. Do you mean I just modified the test?